src/IJulia.jl

```diff
@@ -36,7 +36,7 @@ export notebook, jupyterlab, installkernel
 using ZMQ, JSON, SoftGlobalScope
 import Base.invokelatest
 import Dates
-using Dates: now
+using Dates: now, format, @dateformat_str, UTC, ISODateTimeFormat
```

Comment:
Is `@dateformat_str` needed?

Reply:
Ah! No. 
(In an earlier version of this PR I was manually recreating the `ISODateTimeFormat` with the `'Z'` at the end, but that is no longer needed by the current code.)